code/modules/unit_tests/monkey_business.dm

```diff
@@ -0,0 +1,42 @@
+/**
+ * Monkey Business
+ *
+ * This unit test spawns a predefined number of monkies, each of which
+ * are set to have a 100% chance of attempting to use something next to them each Life
+ *
+ * This test basically just checks to see if attack procs are working correctly,
+ * but its also hilarious and fun to watch locally.
+ */
+/datum/unit_test/monkey_business
+	priority = TEST_DEFAULT + 1 // should be the last test to run, except for create and destroy
```

Comment:
Can you make these defines? `TEST_DEFAULT`, `TEST_AFTER_DEFAULT`, `TEST_CREATE_AND_DESTROY`?